docs/source/configuration.rst

```diff
@@ -0,0 +1,6 @@
+Configuration
+=============
+
+.. jsonschema:: https://raw.githubusercontent.com/jupyterhub/zero-to-jupyterhub-k8s/master/jupyterhub/schema.yaml
```

Comment:
This is me just trying to get anything to print out.

Reply:
I think we should remove the configuration page here in favor of https://github.com/dask/dask/pull/6069 then merge this in

Reply:
Yup.  Thanks for flagging this.  I've removed all of the work around making an actual docpage and now this PR is just about the distributed-schema.yaml file, and testing its accuracy against the config file.  We don't do anything with documentation.